src/index.js

```diff
@@ -58,6 +62,10 @@ class Offline {
             usage: 'The host name to listen on. Default: localhost',
             shortcut: 'o',
           },
+          silent: {
+            usage: 'Silences all serverless offline output. Default: false',
+            shortcut: 's',
```

Comment:
Please do not use a  shortcut for such an option. Plus I think -s may be used by serverless for stage.